src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -1679,8 +1693,9 @@ public static void runActionFromSearchEverywherePanel(RemoteRobot remoteRobot, S
             }
         }
 
-        // Report the last error if there is one.
-        if (error != null) {
+        // Report the last error if there is one and avoid throwing a RuntimeException for the "Close All Tabs" action,
+        // as it is not always necessary for at least one tab to be open in the Editor Window in certain cases.
```

Comment:
Could you improve this comment by describing the problem and the solution? Or explain it in a different way because I do not understand this. (also reported as issue 1152)

Reply:
I updated the comment in this PR. If you think the updated comment is not accurate, we can address issue [#1152](https://github.com/OpenLiberty/liberty-tools-intellij/issues/1152) to revise it further.